lib/l10n/app_en.arb

```diff
@@ -120,6 +120,15 @@
           }
         }
       },
+    "minutes": "{count} min",  
```

Comment:
Can you copy this part to the German and French translation. This should be also valid for those languages.